### DISCUSSION TITLE
GitHub is good!

### DISCUSSION THREAD

**Discussion by arhaze** [community/community#87142] [GitHub Education]
### Hi everyone,

Hi everyone! I am Aries Magumpara an aspiring web developer..

**Comment by ghost** [comment]
Hi @arhaze I too love Github and its products... I am Yuhesh from India, currently interested in Web development and  learning CSS ... Hope this is just your introduction if not sorry for the irrelevant nature of this comment...

Happy Programming Journey

Thank You
